lib/behaviours/session_store.ex

```diff
@@ -21,4 +21,5 @@ defmodule Behaviours.SessionStore do
               {:ok, String.t()} | :error
 
   @callback session_ids_for_user(user_id :: String.t()) :: list()
+  @callback max_ttl(user_id :: String.t()) :: String.t() | nil
```

Comment:
Should this perhaps be `String.t() |  no_return` ?

Reply:
maybe we should return the latest session and not just the ttl? 🤔 then it would be similar to the get operation